cmd/validate/src/lib.rs

```diff
@@ -248,6 +248,7 @@ fn validate(context: &mut ExecutionContext) -> Result<()> {
                     Err(anyhow!("unexpected error type {:?}", op.error))
                 }
             }
+            Err(IpcError::ServerDied(e)) => Ok(e.to_string().red()),
```

Comment:
This will be a bit cryptic, and I don't think the specific dead code is that helpful – maybe just `"server died".red()` instead?

Reply:
gah, whoops, you're right, this was supposed to call `to_string` on the
`IpcError` type, not on the actual ID. will fix.